src/locutus_util/seed_etl/refresh_data.py

```diff
@@ -0,0 +1,164 @@
+"""
+Runs off of the data/seed_etl/seed_config.yml
+For more information, see /locutus_util/seed_etl/README.md
+
+Run examples
+`python seed_etl/refresh_data.py`
+"""
+
+import pandas as pd
+import logging
+from locutus_util.helpers import logger, read_file, parse_owl2_data
+from locutus_util.common import SEED_ETL_DIR
+
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def transform_csv(source_data, output_file, mapping_config):
+    """
+    Transforms a source data into the desired final schema based on the given mapping configuration(seed_config.yaml).
+    Saves the data in the data dir as csv.
+    """
+    try:
+
+        final_schema = [
+            "code", "system", "display", "description",
+            "terminology_id", "terminology_description", "terminology_name", "terminology_resource_type"
```

Comment:
This looks identical to the final_schema below. Maybe this could be a variable that exists outside the scope of either function so you would only change one if there were a need?